### PR TITLE
#154289071  -  Vendor registration

### PR DESCRIPTION
**Pull request by abdulfataiaka**
#### What does this PR do?

- Adds a checkbox with a label that says "I am a vendor" on the registration form
- Shows a modal that tells the user to await shop activations after successful registration
- Sends an email to the user when the shop is activated by the admin

 #### Description of Task to be completed?

-  Edits the registration form to include html checkbox and a label that say "I am a vendor"
-  Writes instructions to create shop and display a modal after shop has been created when the user 
    successfully registers.
-  Uses Reaction.Email.send() method to send email to user regarding new shop activation status 
    when admin activates the shop

 #### How should this be manually tested?

- Clone the repo - `$ git clone https://github.com/andela/k2-rc.git`
- Follow the project README to set up the application on your machine
- run "reaction" command in terminal and navigate to "http://localhost:3000" in your browser
- login as an admin with the credentials below
   -  email : admin@localhost
   -   password: r3@cti0n
- Click on the shop icon on the admin dashboard to show shop settings
- Activate marketplace and enable merchant signup by checking the box provided
- Logout of admin and click the signIn button on the header
- Click the register link on the form that drops down
- Provide valid email and password and click the register button
- A modal pops up telling you to await admin activation of shop upon successful validation.
- Login back as an admin
- Click the marketplace shops on admin dashboard
- Activate the shop listed against the email you registered with earlier
- Check your mail for a message informing you that your new shop has been activated
- Login again with the email and password you provided during registration
- Then on your dashboard, click the dropdown at the top to select the new shop  so your dashboard may reset, revealing functionalities for managing your new shop.

#### What are the relevant pivotal tracker stories?

- story type: feature
- story id: #154289071
- story title: Vendor registration

 ####Any background context you want to provide?

N/A

### CODE REVIEW

File: packages/worker/fileWorker.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ FS.FileWorker.observe = function(fsCollection) {
       try {
         FS.TempStore.removeFile(fsFile);
       } catch(err) {
-        console.log(err);
+        // console.log(err);
```
Comment (damakahindi) [inline packages/worker/fileWorker.js]: Great job on this but why did you comment this out?

File: server/methods/core/shop.js [hunk 2]
```diff
@@ -66,14 +66,18 @@ Meteor.methods({
...
     let shopAccount = currentAccount;
-
     // TODO: Create a grantable permission for creating shops so we can decouple ownership from shop creation
     // Only marketplace owners can create shops for others
     if (hasPrimaryShopOwnerPermission) {
       shopUser = Meteor.users.findOne({ _id: shopAdminUserId }) || currentUser;
       shopAccount = Collections.Accounts.findOne({ _id: shopAdminUserId }) || currentAccount;
     }
 
+    // console.log("\n\n\n\n\n\n\n\n\n\n\n");
```
Comment (damakahindi) [inline server/methods/core/shop.js]: Could you please remove any commented out code in a PR.

File: settings/dev.settings.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "ROOT_URL": "",
-  "MAIL_URL": "",
+  "MAIL_URL": "smtp://abdulfataiaka:fattyolaide22olaidefatty@smtp.gmail.com:465",
```
Comment (damakahindi) [inline settings/dev.settings.json]: @temilaj could you comment on the use of a personal email? Or is this fine for the project?

**Comment by damakahindi** [comment]
Nice work on this PR. Could you fix the changes suggested?

**Comment by abdulfataiaka** [comment]
Ok, making neccessary corrections now

**Comment by abdulfataiaka** [comment]
@damakahindi, Kindly review the requested changes.

**Comment by damakahindi** [comment]
Looks good merging

**Pull request merged**